db/schema.rb

```diff
@@ -932,6 +932,7 @@
     t.datetime "updated_at", null: false
     t.bigint "user_id"
     t.index ["type_of"], name: "index_response_templates_on_type_of"
+    t.index ["user_id", "type_of"], name: "index_response_templates_on_user_id_and_type_of"
```

Comment:
I think the PR is missing the migration file

Reply:
Oh oops 🤦‍♂ I've done that like the past three times lately...